### PR TITLE
feature(gemini): Update SCT to use Gemini 1.8.7 by default.

### PR DESCRIPTION
**Pull request by mykaul**
It was released months ago, we should use it.
Signed-off-by: Yaniv Kaul <yaniv.kaul@scylladb.com>

#### Testing
- [ ]  https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/gemini-3h-test/11/

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: docker/gemini/Dockerfile [hunk 1]
```diff
@@ -5,9 +5,9 @@ WORKDIR /gemini
 RUN apk add curl
 
 ARG version
-ENV version=${version:-1.7.6}
+ENV version=${version:-1.8.7}
 
-RUN curl -LO https://github.com/scylladb/gemini/releases/download/v${version}/gemini_${version}_Linux_x86_64.tar.gz ;\
+RUN curl -LO https://github.com/scylladb/gemini/releases/download/v${version}/gemini_${version}_linux_amd64.tar.gz ;\
```
Comment (mykaul) [inline docker/gemini/Dockerfile:10]: Note the change in the filename. Unsure if we actually use this Dockerfile, btw!

**Comment by fruch** [comment]
I think now gemini should be building the docker on it's own.

so the Dockerfile, are for when we might need a build a specific version

**Comment by mykaul** [comment]
@fruch - unclear what failed in CI. I see this:
```
It was released months ago, we should use it.
[30](https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-6895/3/pipeline-console/?start-byte=0&selected-node=23#log-30)
 Signed-off-by: Yaniv Kaul <yaniv.kaul@scylladb.com>
[31](https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-6895/3/pipeline-console/?start-byte=0&selected-node=23#log-31)
 ✖   subject may not be empty [subject-empty]
[32](https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-6895/3/pipeline-console/?start-byte=0&selected-node=23#log-32)
 ✖   type may not be empty [type-empty]
[33](https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-6895/3/pipeline-console/?start-byte=0&selected-node=23#log-33)
 ✖   scope may not be empty [scope-empty]
```

I'm unsure what they mean.

**Comment by soyacz** [comment]
> @fruch - unclear what failed in CI. I see this:
> 
> ```
> It was released months ago, we should use it.
> [30](https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-6895/3/pipeline-console/?start-byte=0&selected-node=23#log-30)
>  Signed-off-by: Yaniv Kaul <yaniv.kaul@scylladb.com>
> [31](https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-6895/3/pipeline-console/?start-byte=0&selected-node=23#log-31)
>  ✖   subject may not be empty [subject-empty]
> [32](https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-6895/3/pipeline-console/?start-byte=0&selected-node=23#log-32)
>  ✖   type may not be empty [type-empty]
> [33](https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-6895/3/pipeline-console/?start-byte=0&selected-node=23#log-33)
>  ✖   scope may not be empty [scope-empty]
> ```
> 
> I'm unsure what they mean.

your commit should  adhere to `commitlint.config.js` e.g:

```
improvement(gemini): update gemini version to 1.8.7

some commit message describing the change.

```

**Comment by fruch** [comment]
according to the tests, new gemini version isn't working (stuck) if `--schema-seed` is not supplied 

while in
https://github.com/scylladb/scylla-cluster-tests/issues/6355
@dkropachev suggests we should be using this new flag.

so not exactly clear how we should process updating to the newer version

**Comment by mykaul** [comment]
> according to the tests, new gemini version isn't working (stuck) if `--schema-seed` is not supplied
> 
> while in #6355 @dkropachev suggests we should be using this new flag.
> 
> so not exactly clear how we should process updating to the newer version

That's too bad. Makes it dependent on that change, I reckon?

**Comment by fruch** [comment]
> > according to the tests, new gemini version isn't working (stuck) if `--schema-seed` is not supplied
> > while in #6355 @dkropachev suggests we should be using this new flag.
> > so not exactly clear how we should process updating to the newer version
> 
> That's too bad. Makes it dependent on that change, I reckon?

I'm not actually sure if that the intended behavior or not, without @dkropachev clarifying it, we can't really update to this version

**Comment by fruch** [comment]
@dkropachev, we can use newer versions of Gemini, without clearing this (i.e. we can't get any of the fixes you are working on)

**Comment by fruch** [comment]
@roydahan 

we can't update to newer gemini version, with out @dkropachev clarifying the situation,  

> according to the tests, new gemini version isn't working (stuck) if --schema-seed is not supplied

**Comment by fruch** [comment]
@dkropachev 

did you help clarify the situation ? which flags should be used by default ? otherwise we can move to newer version


@roydahan FYI, we forgot about this issue...

**Comment by roydahan** [comment]
Do we have an open issue for it in Gemini?

**Comment by fruch** [comment]
> Do we have an open issue for it in Gemini?
> 

There was only https://github.com/scylladb/scylla-cluster-tests/issues/6355

That ended without anything, and Gemini default behavior if the latest version is broken, without clear advice on how it needs to be configurable.

So no bug fixes (or driver updates) until this is sorted out

**Pull request closed without merging**